turborepo-tests/integration/tests/command-link.t

```diff
@@ -1,6 +1,6 @@
 Setup
   $ . ${TESTDIR}/../../helpers/setup.sh
-  $ . ${TESTDIR}/logged_in.sh
+  $ . ${TESTDIR}/_helpers/logged_in.sh
```

Comment:
nah just convention for "private", and also sorts it at the top of the list of things in the directory so it's easier to find non-test things. Otherwise, I found that existing helpers get missed and duplicated more easily.